CODEOWNERS

```
@@ -0,0 +1,201 @@
+src/handlers/poap.ts @skylarbarrera @osdnk
+src/handlers/svgs.ts @skylarbarrera @osdnk
+src/handlers/tokenSearch.ts @brunobar79 @jinchung @derHowie
+src/handlers/transactions.ts @brunobar79 @osdnk @jinchung
+src/handlers/uniswap.ts @brunobar79 @jinchung
+src/handlers/walletReadyEvents.ts @brunobar79 @jinchung
+src/handlers/web3.ts @brunobar79 @jinchung @estebanmino
+src/handlers/wyre.ts @jinchung @skylarbarrera
+
+# Helpers
+src/helpers @brunobar79 @jinchung
+
+
+# Hooks
+src/hooks @jinchung @osdnk 
+src/hooks/useENSProfile.ts @estebanmino @jxom
+src/hooks/useENSProfileForm.ts @estebanmino @jxom
+src/hooks/useENSRegistration.ts @estebanmino @jxom
+src/hooks/useENSRegistrationCosts.ts @estebanmino @jxom
+src/hooks/useGas.ts @estebanmino @brunobar79 @jinchung
+
+# Languages
+src/languages @skylarbarrera @marco
+
+
+# Model 
+src/model @brunobar79 @jinchung
+src/model/mmkv @osdnk @terrysahaidak
+
+# Navigation 
+src/navigation @osdnk @terrysahaidak
+
+# Parsers 
+src/parsers @brunobar79 @jinchung
+
+# rainbow fetch 
+src/rainbow-fetch @brunobar79 @skylarbarrera
+
+# raps
+src/raps @brunobar79 @jinchung
+src/raps/registerENS.ts @estebanmino @jxom
+
+# charts, shadow stack, cool modals, sheets
+src/react-native-animated-charts @osdnk @terrysahaidak
+src/react-native-cool-modals @osdnk @terrysahaidak
+src/react-native-shadow-stack @osdnk @terrysahaidak
+src/react-native-yet-another-bottom-sheet @osdnk @terrysahaidak
+
+# react query
+src/react-query @jinchung @derHowie
+
+# redux
+src/redux @jinchung @brunobar79
+src/redux/charts.ts @osdnk @terrysahaidak
+src/redux/ensRegistration.ts @estebanmino @jxom
+src/redux/imageMetadata.ts @osdnk @skylarbarrera
+src/redux/gas.ts @brunobar79 @estebanmino
+src/redux/keyboardHeight.ts @osdnk @skylarbarrera
+src/redux/nonceManager.ts @derHowie @brunobar79
+src/redux/showcaseTokens.ts @osdnk @skylarbarrera
+src/redux/transactionSignatures.ts @brunobar79 @estebanmino
+src/redux/uniqueTokens.ts @jinchung @skylarbarrera
+src/redux/walletconnect.js @brunobar79 @estebanmino
+
+# references
+src/references @jinchung @brunobar79 @skylarbarrera
+
+# screens
+src/screens @osdnk @skylarbarrera
+
+# styled-thing
+src/styled-thing @terrysahaidak @osdnk  
+
+# styles
+src/styled-thing @terrysahaidak @osdnk @markdalgleish
+
+# utils
+src/utils @brunobar79 @jinchung
+
+# App
+src/App.ts @brunobar79 @jinchung 
+
+# types
+eth-contract-metadata.d.ts @markdalgleish @jxom
+eth-ens-namehash.d.ts @markdalgleish @jxom
+
+
+# TYPESCRIPT
+types/ @markdalgleish @marco @jinchung
+
+# APP INFRA
+.eslintignore @brunobar79 @osdnk
+.eslintrc.js @brunobar79 @osdnk
+.flowconfig @brunobar79 @osdnk
+.gitattributes @brunobar79 @jinchung
+.gitignore @brunobar79 @jinchung
+.nvmrc @brunobar79 @osdnk
+.prettierrc.js @brunobar79 @osdnk
+.watchmanconfig @brunobar79 @osdnk
+# SECURITY / APP INFRA
+
+.yarnrc @brunobar79
+
+# APP INFRA
+app.json @jinchung @brunobar79
+
+# SECURITY / APP INFRA
+audit-ci.json @brunobar79 @skylarbarrera  
```

```suggestion
audit-ci.json @brunobar79 @skylarbarrera @tchayen
```

I am often fixing CI audits and editing this file from time to time. I'd add myself here.

---

hey! codeowners file just means who is required to review changes to this file - you'll still be able to PR edits to it!